### DISCUSSION TITLE
Unmaintained implementations

### DISCUSSION THREAD

**Discussion by MarmaladeSky** [realworld-apps/realworld#1546] [Community]
Hi there, 
Could anybody please tell how this list is build, and what makes the implementation to be marked as "Unmaintained"?
![image](https://github.com/gothinkster/realworld/assets/2063519/e21434cf-2ef7-4405-b9fa-2dff9634c2b9)

**Comment by geromegrignon** [comment]
Hi, that's based on the commit history as the dependencies API might have changed meanwhile in a way it does not match the modern practices.

It's quite arbitrary as we can't maintain an up to date knowledge of all existing libraries.

@mvila can you remove 'cats' implementation from the unmaintained backend list?

**Comment by mvila** [comment]
@geromegrignon Done.

**Comment by MarmaladeSky** [comment]
Many thanks!
